### PR TITLE
Bugfix/backspace interference with keyboard

### PR DESCRIPTION
**Pull request by saadataz**
- Remove backspace from Direction.BACK to prevent it from interfering in chat
- Added key maps for better readability

### CODE REVIEW

File: src/keymap.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    GamepadRightThumbstick = 210,
+    GamepadLeftThumbstickUp = 211,
+    GamepadLeftThumbstickDown = 212,
+    GamepadLeftThumbstickRight = 213,
+    GamepadLeftThumbstickLeft = 214,
+    GamepadRightThumbstickUp = 215,
+    GamepadRightThumbstickDown = 216,
+    GamepadRightThumbstickRight = 217,
+    GamepadRightThumbstickLeft = 218,
+}
```
Comment (connor4312) [inline src/keymap.ts]: Can we make this its own module or import a dependency for this? We need to do the same sort of thing on frontend and ui

Comment (saadataz) [inline src/keymap.ts]: you can do import keys from 'arcade-machine'

Comment (saadataz) [inline src/keymap.ts]: + frontend and UI both use arcade machine already

**Comment by saadataz** [comment]
[[Chat] X to backspace is closing stream](https://trello.com/c/YzayFmqe/325-chat-x-to-backspace-is-closing-stream)

File: src/keymap.ts (hunk 1 above)
Comment (connor4312) [inline src/keymap.ts]: The desktop site doesn't use arcade-machine yet

Comment (saadataz) [inline src/keymap.ts]: any suggestions where we wanna keep this?

Comment (connor4312) [inline src/keymap.ts]: Using an existing library or making a small 1st part dependency like I did for [durations](https://github.com/WatchBeam/human-duration) would probably work

**Pull request merged**